examples/mongodbatlas_federated_settings_org_role_mapping/Readme.md

```diff
@@ -27,33 +27,37 @@ Now create **terraform.tfvars** file with all the variable values and make sure
 Execute the below command and ensure you are happy with the plan.
 
 ``` bash
-$ terraform plan
+terraform plan
 ```
 This project currently does the below deployments:
 
 - MongoDB Atlas Federated Settings Organizational Role Mapping
-- MongoDB Atlas Federated Settings Organizational Identity Provider
+- MongoDB Atlas Federated Settings Organizational Identity Provider SAML
+- MongoDB Atlas Federated Settings Organizational Identity Provider OIDC
 - MongoDB Atlas Federated Settings Organizational configuration
 
 **4\. Execute the Terraform import for 2 resources that do not support create.**
-``` bash
-$ terraform import mongodbatlas_federated_settings_identity_provider.identity_provider  6287a67f7f7f7f7f441c6c-0oad7f7f7f7fk1297
-  terraform import mongodbatlas_federated_settings_org_config.org_connections_import  6287a67f7f7f7f7f441c6c-627a96837f7f7f7f7e306f14
 
+- find `saml_idp_id` in <https://cloud.mongodb.com/v2#/federation/{federation_settings_id}/identityProviders>
```

Comment:
would put that it can be retrieved through terraform `mongodbatlas_federated_settings_identity_provider` resource & datasource. Also, `saml_idp_id` is not an attribute right? Are you referring to the `idp_id` of the SAML Idp?

Reply:
Good point, renamed to `idp_id`